CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning Scheme](http://semver.org)
 
 ### Changes
```

Comment:
Does this imply "breaking changes"?

Reply:
The previous version has a non-breaking change listed under the same heading. Should be be more explicit about what's breaking?

Also, how do we indicate that the `UNRELEASED` version needs a major version bump?

Reply:
I don't reckon it does imply "breaking changes". Whether we're more explicit or not doesn't bother me much, but I'd be happy to do it if we settle on a convention, and personally I think checking whether you need a major version bump or not at release time is sufficient, but leaving a note in the changelog would also be fine if you'd like?

Reply:
We could do two headings? `Breaking Changes` and `Nonbreaking Changes`? Or note things as (breaking) or (nonbreaking)?